olca-app-html/src/report/charts.tsx

```diff
@@ -154,7 +154,7 @@ export const ComparisonChart = (props: CompProps) => {
     }
 
     if (type === "radar") {
-        config.options.scale = {
```

Comment:
`config.options.scale` is correct.  Otherwise the radard chart is not correctly
displayed. I know `scales` fixes the compile error but the reason for this is
that the type declarations of `@types/chart.js` are not fully complere here.
This can be fixed by changing the `scale` field in the `ChartOptions` interface
in the `@types/chart.js` module to this:

```typescript
// ...
interface ChartOptions {
  // ...
  scale?: ChartScales;
```

Thanks for the pull request though. If you revert this change I would merge it
into the master.

Reply:
perfect. 8ff0c29f compiles fine ;)